Tasks/AzureCLI/task.json

```diff
@@ -79,7 +79,10 @@
             "label": "Arguments",
             "defaultValue": "",
             "required": false,
-            "helpMarkDown": "Arguments passed to the script"
+            "helpMarkDown": "Arguments passed to the script",
+            "properties": {
+                "editorExtension": "ms.vss-services-azure.parameters-grid"
```

Comment:
Should it be this grid. this works only for powershell. We add extra minus sign before parameters in some cases. Confirm with Usha again once